static/src/javascripts/projects/common/modules/analytics/headlines-test-analytics.js

```diff
@@ -0,0 +1,40 @@
+define([
+    'common/utils/detect',
+    'common/utils/config',
+    'common/modules/analytics/beacon'
+], function (
+    detect,
+    config,
+    beacon
+) {
+
+    function clearHash() {
+        if (detect.hasPushStateSupport()) {
+            history.replaceState('', document.title, window.location.pathname);
+        } else {
+            window.location.hash = '';
+        }
+    }
+
+    return {
+        go: function () {
+            var hash = window.location.hash;
+
+            if (window.location.pathname === '/us') {
+                if (config.tests.headlinesAbVariant) {
+                    beacon.counts('headlines-variant-seen');
+                } else if (config.tests.headlinesAbControl) {
+                    beacon.counts('headlines-control-seen');
+                }
+            }
+
+            if (hash === '#headline-variant') {
+                beacon.counts('headlines-variant-clicked');
```

Comment:
Minor note, but I would prefer it if we didn't spam the browser history. The hash means nothing to the user, but it will appear in their history.

![image](https://cloud.githubusercontent.com/assets/921609/9134193/d407c654-3cf9-11e5-9d98-6866eb4e1ca0.png)

Why not use local storage to keep this state, and reset it afterwards? This is fine, but maybe an idea for next time. :+1: